packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -216,6 +216,7 @@ export type CallAdapterClientState = {
     roleHint?: Role;
     cameraStatus?: 'On' | 'Off';
     videoBackgroundImages?: VideoBackgroundImage[];
+    selectedVideoBackgroundEffect?: VideoBackgroundEffect;
```

Comment:
I don't see it in this PR, but the CallComposite background picker's selected item should rely on this state now so its always consistent

Reply:
I am working on a bug for selected item, will make the change sin that PR https://skype.visualstudio.com/SPOOL/_workitems/edit/3217313